tests/constraints/test_constraints_continuous.py

```diff
@@ -68,6 +68,96 @@ def test_inequality3(campaign, n_iterations, batch_size):
+    run_iterations(campaign, n_iterations, batch_size, add_noise=False)
+    res = campaign.measurements
+    print(res)
+
+    for batch in range(n_iterations):
+        res_batch = res[res["BatchNr"] == batch + 1]
+        assert np.isclose(res_batch["Conti_finite1"].sum(), 0.3)
+
+
+@pytest.mark.parametrize("parameter_names", [["Conti_finite1", "Conti_finite2"]])
+@pytest.mark.parametrize("constraint_names", [["InterConstraint_2"]])
+def test_interpoint_inequality_single_parameter(campaign, n_iterations, batch_size):
+    """Test single parameter interpoint inequality constraint."""
+    run_iterations(campaign, n_iterations, batch_size, add_noise=False)
+    res = campaign.measurements
+    print(res)
+
+    for batch in range(n_iterations):
+        res_batch = res[res["BatchNr"] == batch + 1]
+        assert 2 * res_batch["Conti_finite1"].sum() >= 0.299
+
+
+@pytest.mark.parametrize("parameter_names", [["Conti_finite1", "Conti_finite2"]])
+@pytest.mark.parametrize("constraint_names", [["InterConstraint_3"]])
+def test_interpoint_equality_multiple_parameters(campaign, n_iterations, batch_size):
+    """Test interpoint equality constraint involving multiple parameters."""
+    run_iterations(campaign, n_iterations, batch_size, add_noise=False)
+    res = campaign.measurements
+    print(res)
+
+    for batch in range(n_iterations):
+        res_batch = res[res["BatchNr"] == batch + 1]
+        assert np.isclose(
+            res_batch["Conti_finite1"].sum() + res_batch["Conti_finite2"].sum(), 0.3
+        )
+
+
+@pytest.mark.parametrize("parameter_names", [["Conti_finite1", "Conti_finite2"]])
+@pytest.mark.parametrize("constraint_names", [["InterConstraint_4"]])
+def test_geq_interpoint_inequality_multiple_parameters(
+    campaign, n_iterations, batch_size
+):
+    """Test geq-interpoint inequality constraint involving multiple parameters."""
+    run_iterations(campaign, n_iterations, batch_size, add_noise=False)
+    res = campaign.measurements
+    print(res)
+    for batch in range(n_iterations):
+        res_batch = res[res["BatchNr"] == batch + 1]
+        assert (
+            2 * res_batch["Conti_finite1"].sum() - res_batch["Conti_finite2"].sum()
+            >= 0.299
+        )
+
+
+@pytest.mark.parametrize("parameter_names", [["Conti_finite1", "Conti_finite2"]])
+@pytest.mark.parametrize("constraint_names", [["InterConstraint_5"]])
+def test_leq_interpoint_inequality_multiple_parameters(
+    campaign, n_iterations, batch_size
+):
+    """Test leq-interpoint inequality constraint involving multiple parameters."""
+    run_iterations(campaign, n_iterations, batch_size, add_noise=False)
+    res = campaign.measurements
+    print(res)
+    for batch in range(n_iterations):
+        res_batch = res[res["BatchNr"] == batch + 1]
+        print(campaign.searchspace.constraints)
+        assert (
+            2 * res_batch["Conti_finite1"].sum() - res_batch["Conti_finite2"].sum()
+            <= 0.301
```

Comment:
I'd prefer if tolerance aspects are not hardcoded into the number you write here but in a constant defined at the top of the file and expressions just us `0.3 + TOLERANCE` etc
it has been done in other test files I think

Reply:
Done, also fixed an incorrect test that I spotted in the tests.